commons/src/main/java/io/aiven/kafka/tieredstorage/commons/security/RsaEncryptionProvider.java

```diff
@@ -56,8 +58,8 @@ private RsaEncryptionProvider(final KeyPair rsaKeyPair) {
         this.rsaKeyPair = rsaKeyPair;
     }
 
-    public static RsaEncryptionProvider of(final InputStream rsaPublicKey,
-                                           final InputStream rsaPrivateKey) {
+    public static RsaEncryptionProvider of(final Path rsaPublicKey,
+                                           final Path rsaPrivateKey) {
```

Comment:
Considering https://github.com/aiven/tiered-storage-for-apache-kafka/issues/198, maybe we should keep `InputStream`?

Reply:
Good point. What about keeping Paths here, and leave constructor with InputStream instead? Then we can add another static `of` for getting PEM content directly from config, wdyt?

Reply:
Yeah, makes sense

Reply:
Wasn't actually the constructor, but I kept the InputStream method on RsaKeysReader added a new one for Paths. LMK how it looks now